### DISCUSSION TITLE
Rendering for pixel art

### DISCUSSION THREAD

**Discussion by splashsky** [Annoraaq/grid-engine#150] [General]
As a note for people making pixel art-based games, which I imagine applies to most people using this plugin, all our example config code contains this snippet:

```js
render: {
    antialias: true
}
```

But this is two more lines of code than you need.

```js
pixelArt: true
```

Thanks for coming to my TED Talk. 😄

**Comment by lendsp** [comment]
greatly improved the texts in my game
